src/network/core.rs

```diff
@@ -110,6 +114,16 @@ impl Core {
                 ))
             }
         };
+        let host_veth_mac = match core_utils::CoreUtils::get_interface_address(&host_veth_name) {
+            Ok(addr) => addr,
+            Err(err) => {
+                return Err(std::io::Error::new(
+                    std::io::ErrorKind::Other,
+                    format!("failed to get host veth mac: {}", err),
+                ))
+            }
+        };
+        debug!("Host veth mac: {:?}", host_veth_mac);
```

Comment:
Why do we need the host veth mac?

Reply:
@Luap99 I think @baude needs it for some of his experiments.